### ISSUE TITLE
Is HAGS(﻿Hardware-Accelerated GPU Scheduling) strictly required for this mod to work?

### ISSUE THREAD

**Issue by shenhuanchao** [Nukem9/dlssg-to-fsr3#165]
Asking because I have a rare MacBook + RTX 3060 EGPU setup that can't go above Windows 1903/1909. No HAGS for me since the feature was introduced in Windows build 2004. But as far as I know, FSR3 itself doesn't require HAGS to be on to work. So ﻿if it's just something like a boolean check, would it be possible for a future update to enable the feature while HAGS is turned off? Sorry for my ignorance and thanks for any answer in advance!

**Comment by Gaffer13** [comment]
It's strictly required for Nvidia's frame generation to work, which is used by this particular mod. 

But some other modder could implement FSR 3 directly - then you won't need HAGS, but also probably won't be able to use in-game controls to disable/enable frame generation, as they're meant for Nvidia's frame generation.

**Comment by Weather-OS** [comment]
Yes and no. Realistically someone could also write a hook to fool `sl.interposer` that HAGS is enabled, There's also the Linux + Wine HAGS Patch method that pretty much does the same.

**Comment by ForthtuN** [comment]
> It's strictly required for Nvidia's frame generation to work, which is used by this particular mod.
> 
> But some other modder could implement FSR 3 directly - then you won't need HAGS, but also probably won't be able to use in-game controls to disable/enable frame generation, as they're meant for Nvidia's frame generation.

No, Nvidia frame generation is not used. It's replaced by fsr and just unlocked. Nvidias generation is strictly for 4xxx series

**Comment by artur-graniszewski** [comment]
Streamline requires HAGS, but mods like DLSS Enabler on Nexus spoof that capability for you.

https://www.nexusmods.com/site/mods/757?tab=description
